hopsworks-web/yo/app/scripts/controllers/filemgr/main.js

```diff
@@ -0,0 +1,376 @@
+(function(angular) {
```

Comment:
If I remember correctly this code has been taken from somewhere. The original header should be preserved. Also you should add the dependency to the dependency file.